### PR TITLE
Meson

### PR DESCRIPTION
**Pull request by tarmaciltur**
Se portó todo el proyecto a Meson y se eliminó todo lo relacionado a autotools

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,15 @@ compiler: gcc
...
-    - libtool
-    - intltool
-    - autoconf-archive
-    - gobject-introspection
-    - appstream-glib
+    - gtk3
   script:
-    - ./autogen.sh
-    - make
+    - mkdir build
```
Comment (cjfloss) [inline .travis.yml]: You don't need to create a directory, meson does this automatically.

**Comment by cjfloss** [comment]
executing `ninja -C build meson-nomeolvides-pot` gives me the error

```
ninja: Entering directory `build'
[0/1] Running external command nomeolvides-pot.
src/nomeolvides-configuracion.vala:119: warning: unterminated string literal
src/nomeolvides-configuracion.vala:159: warning: unterminated string literal
xgettext: cannot read /home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui: failed to load external entity "/home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui"

xgettext: cannot read /home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui: failed to load external entity "/home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui"

xgettext: cannot read /home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui: failed to load external entity "/home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui"

xgettext: cannot read /home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui: failed to load external entity "/home/cleiton/Public/temp/nomeolvides/src/nomeolvides-treeview-anios.ui"

xgettext: warning: file 'src/nomeolvides-treeview-anios.ui' extension 'ui' is unknown; will try C
xgettext: error while opening "src/nomeolvides-treeview-anios.ui" for reading: No such file or directory
FAILED: meson-nomeolvides-pot``

File: .travis.yml (hunk 1 above)
Comment (ariktaurendil) [reply .travis.yml]: @cjfloss we are trying to be more explicit here.

**Comment by ariktaurendil** [comment]
@cjfloss good catch, already fixed in 2c54b64

**Pull request merged**